### PR TITLE
Minor refactor

### PR DESCRIPTION
**Pull request by fisker**
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-js/utils.js [hunk 1]
```diff
@@ -978,24 +978,11 @@ function isTSXFile(options) {
...
-    case "es5":
-      if (level === "es5") {
-        return true;
-      }
-    // fallthrough
-    case "none":
-    default:
-      return false;
-  }
+function shouldPrintComma(options, level = "es5") {
```
Comment (sosukesuzuki) [inline src/language-js/utils.js:981]: Can you add JSDoc comments to clarify the type of `level`?

Comment (fisker) [reply src/language-js/utils.js:981]: Sorry, I don't know how to do it, can you help me?

Comment (sosukesuzuki) [reply src/language-js/utils.js:986]: Done https://github.com/prettier/prettier/pull/8752/commits/819d58caff169f321e5aeb2c1e6fbd7adf9c4d9e

Comment (fisker) [reply src/language-js/utils.js:986]: Should it be optional?

Comment (sosukesuzuki) [reply src/language-js/utils.js:986]: Yes! Thank you I missed

**Pull request merged**